kalite/distributed/management/commands/kaserve.py

```diff
@@ -135,13 +140,66 @@ def handle(self, *args, **options):
 
         if options['startuplock']:
             os.unlink(options['startuplock'])
-        
+
         # Now call the proper command
         if not options["production"]:
+
+            if options['use_inotify']:
+                self.inject_inotify_reloader()
+
             call_command("runserver", "%s:%s" % (options["host"], options["port"]))
         else:
             del options["production"]
+            del options['use_inotify']
             sys.stdout.write("To access KA Lite from another connected computer, try the following address(es):\n")
             for addr in get_ip_addresses():
                 sys.stdout.write("\thttp://%s:%s/\n" % (addr, settings.USER_FACING_PORT()))
             call_command("runcherrypyserver", *["%s=%s" % (key,val) for key, val in options.iteritems()])
+
+    def inject_inotify_reloader(self):
+        """A function that replaces django.utils.autoreload.reloader_thread's
+        stat based reloading to a function that uses the Linux inotify API.
+
+        This is meant to be a drop-in replacement for reloader_thread,
+        which means that it must conform to its API (as of Django
+        1.5). That means on every detected code change, it must exit
+        with return code 3. The parent thread should then handle the
+        actual reloading of modules.
+
+        """
+
+        from django.utils import autoreload
+        import pyinotify
```

Comment:
If you're worried about an import error, doesn't it make more sense to use a `try`-`except` construct? And I think you should justify in the code (in a comment) why you're importing here instead of at the top of the file.